pkg/pki/k8scsrpki/k8scsr.go

```diff
@@ -22,8 +22,9 @@ import (
 )
 
 const (
-	DependingCsrAnnotation     string = "banzaicloud.io/csr"
-	IncludeFullChainAnnotation string = "csr.banzaicloud.io/fullchain"
+	DependingCsrAnnotation      string = "banzaicloud.io/csr"
+	IncludeFullChainAnnotation  string = "csr.banzaicloud.io/fullchain"
+	CertManagerSignerNamePrefix string = "clusterissuers.cert-manager.io"
```

Comment:
good job on using a const var here instead of hardcoded value in a local function for better visibility of the code while preventing copy&paste hard-coded value errors in the future.

And I am really surprised that upstream cert-manager doesn't have const like this that we can use, they seem to be using the hardcoded string `clusterissuers.cert-manager.io` everywhere across their implementation and tests